raiden/transfer/state_change.py

```diff
@@ -318,21 +318,6 @@ def token_network_address(self) -> TokenNetworkAddress:
 # FIXME: remove on next breaking release
```

Comment:
Do we still need this comment?

Reply:
This (ContractReceiveRouteClosed) shouldn't be removed if the released is supposed to be backwards-compatible.

Reply:
right, but it is removed in this PR isn't it?

Reply:
That's my point. Depending on what the goals are, this should eventually stay.